### PR TITLE
New tutorial: Using KinematicBody2D

### PR DESCRIPTION
**Pull request by cbscribe**
The goal for this doc is to introduce K2D, show the difference between the two movement methods, and provide some common examples. The hope is that many new user questions can then be answered by pointing them here.

Please review and give feedback:
* Mistakes/typos
* Other/better examples
* Anything I missed

### CODE REVIEW

File: tutorials/physics/using_kinematic_body_2d.rst [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+
+.. image:: img/k2d_compare.gif
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    # using move_and_collide
+    var collision = move_and_collide(velocity * delta)
+    if collision:
+    velocity = velocity.slide(collision.normal)
```
Comment (Noshyaar) [inline tutorials/physics/using_kinematic_body_2d.rst]: indent after if ?

**Comment by skyace65** [comment]
Are you going to add the C# code yourself or do you need someone else to do it?

**Comment by cbscribe** [comment]
I will probably leave the C# for someone else. I haven't had a chance to spend much time with it yet.

**Comment by skyace65** [comment]
Assuming there's nothing weird, I should be able to convert the GDScript to C# fairly soon after this is merged.

**Comment by mhilbrunner** [comment]
Read it when you submitted it and a second time now and didn't spot anything wrong or missing. I'll merge this and we'll fix the stuff I probably have missed later :P @skyace65 feel free to do the C#

**Comment by mhilbrunner** [comment]
Cherrypicked as a2ac20f4bae1cfbb1a7244b803254d0aada5c5cd

**Pull request merged**